toplevel/coqtop.ml

```diff
@@ -621,7 +621,7 @@ let init_toplevel arglist =
       Spawned.init_channels ();
       Envars.set_coqlib ~fail:(fun msg -> CErrors.user_err Pp.(str msg));
       if !print_where then (print_endline(Envars.coqlib ()); exit(exitcode ()));
-      if !print_config then (Envars.print_config stdout; exit (exitcode ()));
+      if !print_config then (Envars.print_config stdout Coq_config.all_src_dirs; exit (exitcode ()));
```

Comment:
I think this should also made to understand the bypass-API flag, or people writing their own Makefiles will have no choice.